Lib/test/test_sysconfig.py

```diff
@@ -415,10 +415,10 @@ def test_user_similar(self):
                 # bpo-44860: platlib of posix_user doesn't use sys.platlibdir,
                 # whereas posix_prefix does.
                 if name == 'platlib':
-                    # Replace "/lib64/python3.11/site-packages" suffix
-                    # with "/lib/python3.11/site-packages".
-                    py_version_short = sysconfig.get_python_version()
-                    suffix = f'python{py_version_short}/site-packages'
+                    # Replace "/lib64/python3.11{abi_thread}/site-packages" suffix
+                    # with "/lib/python3.11{abi_thread}/site-packages".
```

Comment:
Let's keep the example concrete, even at the cost of not covering all the cases.

```suggestion
                    # Replace "/lib64/python3.11/site-packages" suffix
                    # with "/lib/python3.11/site-packages".
```